server/utils/matchUtils.ts

```diff
@@ -363,3 +365,29 @@ export const placementRequestSummaryListForMatching = (placementRequest: Placeme
 
   return rows
 }
+
+export const keyDetails = (placementRequest: PlacementRequestDetail): KeyDetailsArgs => {
+  const { person } = placementRequest
+  if (!isFullPerson(person)) throw Error('Restricted person')
```

Comment:
If the person is restricted, what do we want to happen to this bar? Does the error bubble up so the whole page isn't rendered?

Reply:
It's a good point - I'm not really sure what should happen, it's one of those things that shouldn't happen (but will). In this case there will be a blocking error. 
If the CRN is restricted the user won't be able to click on it on the CRU Dashboard 
![image](https://github.com/user-attachments/assets/6b1f4adc-90bd-4e94-b46b-86206b8fcd69)
Which means they won't see the link which takes them to the search pages which means they shouldn't be on this page.  
In time we will probably want a more helpful error but I feel like that change is out of scope for now.